fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java

```diff
@@ -519,6 +521,12 @@ public class SessionVariable implements Serializable, Writable {
     @VariableMgr.VarAttr(name = ENABLE_FUNCTION_PUSHDOWN)
     public boolean enableFunctionPushdown;
 
+    /**
+     * For debugg purpose, dont' merge unique key and agg key when reading data.
+     */
+    @VariableMgr.VarAttr(name = AGG_AS_DUPLICATE)
+    private boolean aggAsDuplicate = false;
```

Comment:
the variable name： skip_storage_engine_merge

Reply:
done